Cargo.toml

```diff
@@ -134,7 +134,7 @@ crossbeam-queue = { version = "0.3.11" }
 darling = "0.20.10"
 dashmap = "6.1.0"
 deadpool = { version = "0.12.1", features = ["rt_tokio_1"] }
-deadpool-postgres = "0.14.0"
```

Comment:
Also non-blocking, but what is using this feature?

Reply:
PoolConfig derives ser/de so it can be `try_from`'d the command line args. I needed it so we can use `RecycleMethod` there